Changelog.md

```diff
@@ -10,6 +10,7 @@
 * Add `entriesFrom()` and `reverseEntriesFrom()` to `Map`, `valuesFrom()` and `reverseValuesFrom()` to `Set` and `Text.toText()` (#272).
 * Update code examples in doc comments (#224, #282, #303, #315).
 * Add `findIndex()` function to modules with `find()` (#321).
+* Adjust `List` and `pure/List` APIs for consistency (#322).
```

Comment:
This is a breaking change, so I'd add add `**Breaking:** ` prefix to it or sth like that. [Rationale here](https://github.com/vweevers/common-changelog?tab=readme-ov-file#244-prefixes)